lib/posix/getopt/getopt_long.c

```diff
@@ -223,47 +225,51 @@ parse_long_options(struct getopt_state *state, char * const *nargv,
 		 * If this is a known short option, don't allow
 		 * a partial match of a single character.
 		 */
-		if (short_too && current_argv_len == 1)
+		if (short_too && current_argv_len == 1) {
 			continue;
+		}
 
-		if (match == -1)        /* first partial match */
+		if (match == -1) { /* first partial match */
 			match = i;
-		else if ((flags & FLAG_LONGONLY) ||
-			 long_options[i].has_arg !=
-			     long_options[match].has_arg ||
-			 long_options[i].flag != long_options[match].flag ||
-			 long_options[i].val != long_options[match].val)
+		} else if ((flags & FLAG_LONGONLY) ||
+			   long_options[i].has_arg != long_options[match].has_arg ||
+			   long_options[i].flag != long_options[match].flag ||
+			   long_options[i].val != long_options[match].val) {
 			second_partial_match = 1;
```

Comment:
fix formatting.

Reply:
what's wrong with this one?

Reply:
nothing, looking at some many changes at once, didn't realize that the `long_options` are part of the `if`